conf/application.conf

```diff
@@ -59,6 +59,7 @@ microservice {
       online-trade-tariff-api {
         url = "http://localhost:10903"
         bearerToken = "bearerToken"
+        apiKey = "apiKey"
```

Comment:
does any of this need to change for base config, staging config, qa config etc?

Reply:
No changes needed as we want to keep it as a default apiKey in staging, it will be the encrypted value in QA and production which is added through the Jenkins secret builder. Currently has been added to QA, just awaiting on testing before adding it to prod